kyuubi-common/src/main/scala/org/apache/kyuubi/operation/log/LogDivertAppender.scala

```diff
@@ -57,7 +77,25 @@ class LogDivertAppender extends WriterAppender {
 }
 
 object LogDivertAppender {
+  val writer = new CharArrayWriter()
+
+  def initLayout(): StringLayout = {
```

Comment:
Let's algin with `LogDivertAppender12` moves them to the `class`.

The buffer `writer` should be a member variable not a static variable I think. The `layout`, since we only init it one time, move to the `class` would not cause the perf issue

Reply:
Have removed writer into the `class`.
But `initLayout` need be used in constructor, so, if I place it into class, compile error would be thrown.